src/dev/impl/DevToys/Helpers/CertificateHelper.cs

```diff
@@ -68,6 +70,38 @@ internal static bool TryDecodeCertificate(string input, string? password, out st
                 return false;
             }
 
+            // Try to decode the X.509 extensions. Put this is a separate try/catch because it's not critical.
```

Comment:
Should we move this block to a separate (private) method? I think it would improve readability and also would make it so that you could conditionally concatenate to the existing `decoded` value if a non-null/empty string return is handed back from that function.

Reply:
Good idea! Is that what you had in mind?

![image](https://github.com/veler/DevToys/assets/7417301/a374b34d-7e91-4dd8-b7ee-16e66d473614)

Reply:
Yes, that looks good.  I think perhaps the line under the first `if` statement you have there might be suggesting to "return early" - so it would be something like this:
```csharp
if (!certificate.Extensions.OfType<X509Extension>().Any())
{
    return null;
}

try
{
// ... rest of method logic here
```

Reply:
23c2d9cb2d9e9b94d5a356b1893d40a28cc1c578